guides/source/getting_started.md

```diff
@@ -492,7 +492,7 @@ Go ahead now and create a new file at `app/views/articles/new.html.erb` and
 write this content in it:
 
 ```html
-<h1>New Article</h1>
+<h1>New article</h1>
```

Comment:
This change might be legitimate, but it’s unrelated to the image correction. Either this PR should be titled and described like a set of multiple corrections to the Getting Started guide or the text correction belongs in a separate PR. It’s small enough that the former is probably best.